### PR TITLE
Unshare TokenAccountIndividual objects

### PR DESCRIPTION
**Pull request by cjavilla-stripe**
- [x] ⚠️ Unshare `TokenAccountIndividual` objects

r? @remi-stripe 
cc @stripe/api-libraries

### CODE REVIEW

File: src/Stripe.net/Services/Accounts/AccountCreateOptions.cs [hunk 1]
```diff
@@ -7,123 +7,50 @@ namespace Stripe
...
         [JsonConverter(typeof(AnyOfConverter))]
         public AnyOf<string, AccountBankAccountOptions, AccountCardOptions> ExternalAccount { get; set; }
 
-        /// <summary>
-        /// Information about the person represented by the account. This field
-        /// is null unless <see cref="BusinessType"/> is set to <c>individual</c>.
-        /// </summary>
         [JsonProperty("individual")]
-        public PersonCreateOptions Individual { get; set; }
+        public AccountIndividualOptions Individual { get; set; }
```
Comment (remi-stripe) [inline src/Stripe.net/Services/Accounts/AccountCreateOptions.cs:39]: Did you forget to rebase? This was in the previous PR we already merged https://github.com/stripe/stripe-dotnet/pull/2159/files#diff-b1db3acda2c6906eb4ad36e4d5e8d7acR39

**Comment by cjavilla-stripe** [comment]
Yep the rebase didn't happen. Soooory :) Thanks for the second look

File: src/Stripe.net/Services/Tokens/TokenAccountIndividualOptions.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        public string Phone { get; set; }
+
+        [JsonProperty("political_exposure")]
+        public string PoliticalExposure { get; set; }
+
+        [JsonProperty("ssn_last_4")]
+        public string SsnLast4 { get; set; }
+
+        [JsonProperty("verification")]
+        public TokenPersonVerificationOptions Verification { get; set; }
```
Comment (remi-stripe) [inline src/Stripe.net/Services/Tokens/TokenAccountIndividualOptions.cs:63]: This one is still wrong I think

File: src/Stripe.net/Services/Tokens/TokenAccountIndividualVerificationOptions.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class TokenAccountIndividualVerificationOptions : INestedOptions
+    {
+        [JsonProperty("additional_document")]
+        public TokenPersonVerificationDocumentOptions AdditionalDocument { get; set; }
```
Comment (remi-stripe) [inline src/Stripe.net/Services/Tokens/TokenAccountIndividualVerificationOptions.cs:8]: same here, name is wrong

File: src/Stripe.net/Services/Tokens/TokenAccountIndividualOptions.cs (hunk 2 above)
Comment (cjavilla-stripe) [reply src/Stripe.net/Services/Tokens/TokenAccountIndividualOptions.cs:63]: hmm. good catch. looking

**Comment by cjavilla-stripe** [comment]
okay updated ptal @remi-stripe

**Pull request merged**